app/Http/Controllers/Admin/QuizResponseController.php

```diff
@@ -44,18 +43,14 @@ public function index(Request $request, ?Quiz $quiz = null)
     public function store(Event $event, Team $team)
     {
         if (!$event->active_quiz_id) {
```

Comment:
```suggestion
        if (! $event->active_quiz_id) {
```
Are we not aiming consistent linting here?